examples/deployment/deployment.tf

```diff
@@ -21,19 +21,11 @@ resource "ec_deployment" "example_minimal" {
   version                = "7.9.2"
   deployment_template_id = "aws-io-optimized-v2"
 
-  elasticsearch {}
-
-  kibana {}
-
-  apm {
-    topology {
-      size = "0.5g"
+  elasticsearch {
+    config {
+	user_settings_yaml = file("./es_settings.yaml")
     }
   }
 
-  enterprise_search {
-    topology {
-      zone_count = 1
-    }
-  }
```

Comment:
The idea with these two was to show the different blocks there are for a deployment resource and how to use the topology element block. I think it's a good idea to keep them. I'm not sure what happened before but I tried the example myself and all the resources were created successfully :)

Reply:
Let me re-test this.